hazelcast/src/main/java/com/hazelcast/jet/impl/operation/SubmitJobOperation.java

```diff
@@ -33,24 +34,31 @@
     private Data serializedConfig;
     private boolean isLightJob;
 
+    private transient Subject subject;
```

Comment:
Why is the `subject` transient? With non-smart client, the message can land on non-master member and will be forwarded to the master, the subject will be lost and no permissions will be checked.